### PR TITLE
Create [HuberStefan].md

### PR DESCRIPTION
**Pull request by HuberStefan**
Test

### CODE REVIEW

File: docs/_slides/[HuberStefan].md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from [username]!
```
Comment (HuberStefan) [inline docs/_slides/[HuberStefan].md:6]: Hey @kieblerto passt das so für dich?

**Review by kieblerto — CHANGES_REQUESTED** [review]

File: docs/_slides/[HuberStefan].md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+![dinotocat](https://octodex.github.com/images/dinotocat.png)
```
Comment (kieblerto) [inline docs/_slides/[HuberStefan].md:8]:
```suggestion
![dinotocat](https://octodex.github.com/images/saint_nictocat.jpg)
```

It's time for christmas

**Review by HuberStefan — COMMENTED** [review]
Nice suggestion

**Review by kieblerto — APPROVED** [review]

File: docs/_slides/[HuberStefan].md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from [username]!
+
+![dinotocat](https://octodex.github.com/images/saint_nictocat.jpg)
+{: .center}
```
Comment (kieblerto) [inline docs/_slides/[HuberStefan].md:9]: no new line at EOF

**Review by kieblerto — CHANGES_REQUESTED** [review]

**Review by kieblerto — APPROVED** [review]

**Pull request merged**